exports/sharedAbis.json

```diff
@@ -1,224 +1,290 @@
 {
-    "ERC20": [
-        {
-            "constant": true,
-            "inputs": [],
-            "name": "name",
-            "outputs": [
-                {
-                    "name": "",
-                    "type": "string"
-                }
-            ],
-            "payable": false,
-            "stateMutability": "view",
-            "type": "function"
+  "ERC20": [
```

Comment:
I think we've already discussed something like this,so sorry...
I have a bit different question now, why cannot we use ERC20 artifact which is being created in the `build` folder each time after we run compile?

Reply:
Thanks, good point, that works. I'll remove the integration in the hardhat config.